js/core/devices.js

```diff
@@ -224,8 +224,12 @@ var Devices = Class.inherit({
             this._forced = true;
             this.changed.fire();
 
-            if(this._currentDevice.platform === "win" && this._currentDevice.version[0] === 8) {
-                errors.log("W0010", "the 'win8' theme", "16.1", "Use the 'win10' theme instead.");
+            if(this._currentDevice.platform === "win") {
+                if(this._currentDevice.version[0] === 8) {
+                    errors.log("W0010", "The 'win8' theme", "16.1", "Use the 'generic' theme instead.");
```

Comment:
Could we possibly reword messages to not use the word "theme"? It's a bit foreign in the "devices" module.
Alternatively, could this warning be transferred to another module?

/cc @RomanTsukanov @viterobk

Reply:
@AlekseyMartynov 

I think that current messages reflect our intentions. One thing that can be improved is moving this messages to themes module. But I suggest doing this in a context of another PR that will be added to next DevExtreme version. The current solution works well since 16.1 and we do not want to destabilize the 17.2.3 build now.
destabilize

Reply:
I'm going to move these warnings to themes module asap.